services/OrderService.py

```
@@ -0,0 +1,130 @@
+"""
```

Pylint (3.3.7)

[convention] Module name "OrderService" doesn't conform to snake_case naming style

(C0103)